src/Handler/FormErrorHandler.php

```diff
@@ -35,7 +35,7 @@ public static function getSubscribingMethods()
         foreach (['xml', 'json'] as $format) {
             $methods[] = [
                 'direction' => GraphNavigatorInterface::DIRECTION_SERIALIZATION,
-                'type' => Form::class,
+                'type' => FormInterface::class,
```

Comment:
can you please add `FormInterface` as handled type instead of replacing the `Form::class`. I'm pretty sure that there could be side effects to this change :)

Reply:
done